### PR TITLE
Fix docker build

### PR DESCRIPTION
**Pull request by gemmai95**
Fix docker build.

## Checklist
  - [ ] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by gemmai95** [comment]
[(3) Remove sdx common clone in Dockerfiles](https://trello.com/c/AfHeYs4M/400-3-remove-sdx-common-clone-in-dockerfiles)

File: requirements.txt [hunk 1]
```diff
@@ -5,5 +5,5 @@ gunicorn==19.6.0
 pika==0.10.0
 PyJWT==1.4.2
 requests==2.12.4
-sdx-common==0.7.0
+sdx-common==0.7.2
```
Comment (necrophonic) [inline requirements.txt:8]: Should all these requirements be hashed in line with other repos?

Comment (gemmai95) [reply requirements.txt:8]: Should they be hashed on the master or 2.0?

Comment (necrophonic) [reply requirements.txt:8]: Certainly in the 2.0 version - if this is the old version then it's not so vital as it'll go away anyhow.

**Pull request closed without merging**